api/content/data/front.rng

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<grammar ns="http://libero.pub" xmlns="http://relaxng.org/ns/structure/1.0"
+         xmlns:a="http://relaxng.org/ns/compatibility/annotations/1.0">
+
+    <include href="../../../core/data/core.rng"/>
+
+    <start>
+        <ref name="libero.front"/>
+    </start>
+
+    <div>
+
+        <a:documentation>front element</a:documentation>
+
+        <define name="libero.front">
+
+            <element name="front">
+                <ref name="libero.front.attributes"/>
+                <ref name="libero.front.content"/>
+            </element>
+
+        </define>
+
+        <define name="libero.front.attributes">
+            <ref name="libero.attributes.lang"/>
+        </define>
+
+        <define name="libero.front.content">
+
+            <interleave>
+                <ref name="libero.front.content.id"/>
+                <ref name="libero.front.content.title"/>
+            </interleave>
+
+        </define>
+
+    </div>
+
+    <div>
+
+        <a:documentation>front/id element</a:documentation>
+
+        <define name="libero.front.content.id">
+
+            <element name="id">
+                <ref name="libero.front.content.id.attributes"/>
+                <ref name="libero.front.content.id.content"/>
+            </element>
+
+        </define>
+
+        <define name="libero.front.content.id.attributes">
+            <empty/>
+        </define>
+
+        <define name="libero.front.content.id.content">
+            <ref name="libero.types.id"/>
+        </define>
+
+    </div>
+
+    <div>
+
+        <a:documentation>front/title element</a:documentation>
+
+        <define name="libero.front.content.title">
+
+            <element name="title">
+                <ref name="libero.front.content.title.attributes"/>
+                <ref name="libero.front.content.title.content"/>
+            </element>
+
+        </define>
+
+        <define name="libero.front.content.title.attributes">
+            <empty/>
+        </define>
+
+        <define name="libero.front.content.title.content">
+            <ref name="libero.text.limited"/>
```

Comment:
Design decision: how to handle subtitles. [JATS uses separate elements](https://jats.nlm.nih.gov/archiving/tag-library/1.1/element/title-group.html), [TEI inlines it](http://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-title.html).